web/mailer.ex

```diff
@@ -3,9 +3,6 @@ defmodule HexWeb.Mailer do
     assigns = [layout: {HexWeb.EmailView, "layout.html"}] ++ assigns
     body    = Phoenix.View.render(HexWeb.EmailView, template, assigns)
 
-    HexWeb.Utils.task(fn ->
-      fun = fn email -> HexWeb.Email.send([email], title, body) end
-      HexWeb.Parallel.run(fun, emails)
-    end)
+    HexWeb.Email.send(emails, title, body)
```

Comment:
We probably have to chunk on the throttle rate here and send multiple emails or we will crash if we send a mail to more than 14 recipients?